### PR TITLE
28.0.2 RC1, 27.1.6 RC1, 26.0.11 RC1

### PR DESCRIPTION
**Pull request by blizzz**

### CODE REVIEW

File: tests/integration/features/beta.feature [hunk 1]
```diff
@@ -691,23 +691,23 @@ Feature: Testing the update scenario of beta releases
     And the installation mtime is "11"
     When The request is sent
     Then The response is non-empty
-    And Update to version "26.0.10.1" is available
-    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-26.0.10.zip"
+    And Update to version "26.0.11.0" is available
+    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-26.0.11rc1.tar.bz2"
```
Comment (nickvergessen) [inline tests/integration/features/beta.feature:695]: zip?

Comment (blizzz) [reply tests/integration/features/beta.feature:695]: aye

File: tests/integration/features/beta.feature [hunk 2]
```diff
@@ -670,18 +670,18 @@ Feature: Testing the update scenario of beta releases
...
```
Comment (nickvergessen) [inline tests/integration/features/beta.feature:674]: zip

File: config/config.php [hunk 3]
```diff
@@ -699,98 +699,98 @@
...
 			],
 		],
 		'26' => [
 			'100' => [
-				'latest' => '26.0.10',
-				'internalVersion' => '26.0.10.1',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-26.0.10.zip',
+				'latest' => '26.0.11 RC1',
+				'internalVersion' => '26.0.11.0',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.11rc1.tar.bz2',
```
Comment (nickvergessen) [inline config/config.php:768]:
```suggestion
				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-26.0.11rc1.zip',
```

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**